config.js

```diff
@@ -4,7 +4,6 @@ System.config({
   babelOptions: {
     "optional": [
       "runtime",
-      "optimisation.modules.system",
```

Comment:
leftover from previous try. didn't help. but i had a problem with that line somewhere else, 
...
had a quick look now, and i clearly should have left it there